core/NameHash.h

```diff
@@ -44,7 +44,8 @@ struct GlobalStateHash {
     static constexpr int HASH_STATE_INVALID = 2;
     static constexpr int HASH_STATE_INVALID_COLLISION_AVOID = 3;
     u4 hierarchyHash = HASH_STATE_NOT_COMPUTED;
-    UnorderedMap<NameHash, u4> methodHashes;
+    // N.B.: methodHashes is sorted.
```

Comment:
Can we delete this comment and replace it with an ENFORCE somewhere?

Reply:
For example, one option would be to make a helper method instead of exposing the field directly, and ENFORCE(is_sorted) in that helper.

Reply:
alternatively, you could ENFORCE at the usage site(s).

Reply:
I ENFORCEd it at the usage site.